public/app/features/dashboard-scene/utils/utils.ts

```diff
@@ -200,35 +200,44 @@ export function isPanelClone(key: string) {
   return key.includes('clone');
 }
 
-export function getNextPanelId(dashboard: DashboardScene) {
+export function getNextPanelId(dashboard: DashboardScene): number {
```

Comment:
Wondering if this would sit better in `dashboardSceneGraph` ?

Reply:
I'd say yes! 👍